scripts/quests/crystalWar/WOTG_BAS_0_The_Fighting_Fourth.lua

```diff
@@ -0,0 +1,267 @@
+-----------------------------------
+-- The Fighting Fourth (WOTG Nation Quests Bastok Intro)
+-- !addquest 7 7
+-- Turbulent Storm : !pos 422.461 -48.000 -47.308 175
+-- Adelbrecht      : !pos -325.766 -12.601 -76.977 87
+-- Gebhardt        : !pos 206.299 -20.826 669.125 88
+-- Roderich        : !pos -400.039 39.991 -90.445 88
+-- Barricade       : !pos -514.960 37.979 583.287 88
+-----------------------------------
+local marketsID = zones[xi.zone.BASTOK_MARKETS_S]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.CRYSTAL_WAR, xi.quest.id.crystalWar.THE_FIGHTING_FOURTH)
+
+quest.reward =
+{
+    -- TODO: The messaging for this should be first keyItem and then item, but is swapped
```

Comment:
we could check for inv space > 0 then add ki and quest:complete() to rectify this, janky but removes the todo